packages/trackerless-network/src/logic/DiscoveryLayerNode.ts

```diff
@@ -1,4 +1,13 @@
-import { DhtAddress, PeerDescriptor, RingContacts } from '@streamr/dht'
+import { DhtAddress, PeerDescriptor, RingContacts, ServiceID } from '@streamr/dht'
+import { StreamPartID } from '@streamr/utils'
+
+export const DEFAULT_DISCOVERY_LAYER_KBUCKET_SIZE = 4
+export const DEFAULT_DISCOVERY_LAYER_JOIN_TIMEOUT = 20000
+export const DEFAULT_DISCOVERY_LAYER_NEIGHBOR_PING_LIMIT = 16
```

Comment:
What this limits? E.g. the count of neighbors to ping, or interval of pings, or timeout of ping responses?

Reply:
It limits pings if the number of neighbors is above it. Maybe it could be called ie `DEFAULT_DISCOVERY_LAYER_NEIGHBOR_COUNT_PING_LIMIT`?